File: examples/installation/osx/objc/CarthageExample/CarthageExampleTests/CarthageExampleTests.m
//
//  CarthageExampleTests.m
//  CarthageExampleTests
//
//  Created by JP Simard on 5/11/15.
//  Copyright (c) 2015 Realm. All rights reserved.
//

#import <Cocoa/Cocoa.h>
#import <XCTest/XCTest.h>
#import "AppDelegate.h"

@interface CarthageExampleTests : XCTestCase
@end

@implementation CarthageExampleTests

- (void)testExample {
    XCTAssertTrue([MyModel isSubclassOfClass:[RLMObject class]]);
}

@end

File: examples/installation/osx/swift-2.1/CarthageExample/CarthageExampleTests/CarthageExampleTests.swift
//
//  CarthageExampleTests.swift
//  CarthageExampleTests
//
//  Created by JP Simard on 6/25/15.
//  Copyright (c) 2015 Realm. All rights reserved.
//

import Cocoa
import XCTest
import CarthageExample
import RealmSwift

class CarthageExampleTests: XCTestCase {
    func testExample() {
        // This is an example of a functional test case.
        XCTAssertNotNil(MyModel() as AnyObject is Object)
    }
}
